java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -61,7 +61,7 @@
 
 extern "C" {
 
-JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_upperStrings(JNIEnv *env, jobject j_object,
```

Comment:
I'd like to see this renamed to `ColumnViewJni.cpp` to line up with the corresponding Java file as the other Jni.cpp files do.  JNI methods specific to a ColumnVector, like making a `column_view` from a `column` would remain here.

Reply:
I renamed this file and have moved column methods to the new shorter ColumnVectorJni.